core/src/main/java/org/apache/druid/java/util/metrics/JvmMonitor.java

```diff
@@ -74,10 +77,19 @@ public boolean doMonitor(ServiceEmitter emitter)
     emitJvmMemMetrics(emitter);
     emitDirectMemMetrics(emitter);
     emitGcMetrics(emitter);
+    emitThreadAllocationMetrics(emitter);
 
     return true;
   }
 
+  private void emitThreadAllocationMetrics(ServiceEmitter emitter)
+  {
+    final ServiceMetricEvent.Builder builder = builder();
+    MonitorUtils.addDimensionsToBuilder(builder, dimensions);
+    Optional<Long> delta = collector.calculateDelta();
+    delta.ifPresent((value) -> emitter.emit(builder.build("jvm/heapAlloc/bytes", value)));
```

Comment:
Maybe `jvm/heapAlloc/bytes/delta`

Reply:
I'd say `.../delta/bytes`

Reply:
Here: #6425 "delta" is the last part, what are the statistics across all metrics?

Reply:
What statistics? Just checked the code and `/delta/*` matches > 50 and `/delta"` just half a dozen.

Reply:
Yes, I meant the statistics that you provided. Let it be delta/bytes then